backend/reservations/reporting.py

```diff
@@ -139,3 +139,76 @@ def hotel_export(hotel):
 
     egest_csv(rows, fields, hotel_export_file)
     return hotel_export_file
+
+
+def rooming_list_export(hotel):
+    rooms = Room.objects.filter(name_hotel = hotel.title())
+    guests = Guest.objects.filter(name_hotel = hotel.title())
```

Comment:
Oh snap missed this earlier. In `Guest` this field is just called `hotel` because I guess we didn't boof enough UML or something.
```suggestion
    guests = Guest.objects.filter(hotel = hotel.title())
```

Reply:
It's not being used, since we have the guest reference on the room. removing.